data/campaigns/Heir_To_The_Throne/scenarios/17_Scepter_of_Fire.cfg

```diff
@@ -88,6 +90,9 @@
                     [/gold_carryover]
 
                     [note]
+                        [show_if]
+                            {VARIABLE_CONDITIONAL moved_too_close boolean_equals false}
+                        [/show_if]
```

Comment:
Re-tested for the [objectives] question about [show_if]

The objectives do not display when the condition changes (as desired).

The dependent variable changes when Delfador makes his comment that the Sceptre is near.

If the user then asks for the objectives, the note has disappeared (as desired).

So, no, when using [show_if] you do not need to execute an [objectives] action.

Which causes me to re-think my other changes for this. I'm probably doing the [objectives] action way too often and can make better use of [show_if]. I will examine that issue tomorrow.